### PR TITLE
sci-libs/proj: version bump to 5.2.0

### PR DESCRIPTION
**Pull request by cjmayo**
Drop Java support:

- Requires at least JDK 9 which is not available from a Gentoo supported VM
- ebuild needs changing to use ant

Bug: https://bugs.gentoo.org/622916
Bug: https://bugs.gentoo.org/623164
Bug: https://bugs.gentoo.org/623640

---

sci-libs/proj: update longdescription in metadata.xml

Rebranded as PROJ with greater scope.

---

### CODE REVIEW

File: sci-libs/proj/proj-5.2.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+src_install() {
+	default
+	cd nad || die
+	dodoc README.{NAD,DATUMGRID}
+	insinto /usr/share/proj
+	insopts -m 755
+	doins test27 test83
+	insopts -m 644
+	doins pj_out27.dist pj_out83.dist
+	find "${D}" -name '*.la' -delete || die
```
Comment (monsieurp) [inline sci-libs/proj/proj-5.2.0.ebuild]: Please call `prune_libtool_files` instead.

Comment (juippis) [reply sci-libs/proj/proj-5.2.0.ebuild]: Don't use it. 

https://devmanual.gentoo.org/eclass-reference/ltprune.eclass/index.html

it was purposely updated to die with EAPI-7 so people would stop using it. Which reminds me, this ebuild should use EAPI-7.

**Comment by cjmayo** [comment]
Updated to EAPI 7. I thought there was potentially some problem if Java was reinstated, but seems OK.

On libtool just let me know.

File: sci-libs/proj/proj-5.2.0.ebuild [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+	http://download.osgeo.org/proj/${P}.tar.gz
+	http://download.osgeo.org/proj/${DATUMGRID}
+"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~arm64 ~ia64 ~ppc ~ppc64 ~s390 ~sparc ~x86 ~amd64-linux ~x86-linux ~ppc-macos ~x86-macos"
+IUSE="static-libs"
+
+RDEPEND="${DEPEND}"
```
Comment (juippis) [inline sci-libs/proj/proj-5.2.0.ebuild]: This is rather useless, but LGTM otherwise! :)

Comment (cjmayo) [reply sci-libs/proj/proj-5.2.0.ebuild]: Removed. Didn't make much sense but it was in skel.ebuild until last week 7d0500007c.

**Comment by titanofold** [comment]
The latest version is now 6.1.0, and we do have a virtual/jdk-11 in the tree, albeit masked.

**Comment by cjmayo** [comment]
> The latest version is now 6.1.0, and we do have a virtual/jdk-11 in the tree, albeit masked.

Yes but the Java support in the ebuilds currently in the repository don't actually install the .jar file.

Someone would need to fix the Java support. I did have that working for proj 5.0.1 and have posted an untested ebuild on https://bugs.gentoo.org/685224.

**Comment by Amynka** [comment]
Since there is version 6.0.0 is this PR still valid? If yes can you please rebase the manifest changes are outdated.

**Comment by cjmayo** [comment]
PROJ 6.x changes the API so a lesser version needed while there are dependent ebuilds in the repository. The above linked bugs are security issues for 4.x.
(N.B. 6.0.0 ebuild has several issues https://github.com/gentoo/gentoo/pull/11963#issue-277883836)

Rebased on master.

**Pull request closed without merging**